File: src/main/java/com/telegram/folobot/repos/FoloUserRepo.java
package com.telegram.folobot.repos;

import com.telegram.folobot.domain.FoloUserEntity;
import lombok.NonNull;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface FoloUserRepo extends CrudRepository<FoloUserEntity, Long> {
    @NonNull List<FoloUserEntity> findAll();
}

File: src/main/kotlin/com/telegram/folobot/persistence/repos/FoloUserRepo.kt
package com.telegram.folobot.persistence.repos

import com.telegram.folobot.persistence.entity.FoloUserEntity
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface FoloUserRepo : CrudRepository<FoloUserEntity, Long> {
    fun findUserByUserId(userId: Long): FoloUserEntity?
}